torch/csrc/tensor/python_tensor.cpp

```diff
@@ -252,6 +237,17 @@ static void initialize_aten_types(std::vector<PyTensorType>& tensor_types) {
   }
 }
 
+static PyTensorType& get_tensor_type(THPDtype *dtype, THPLayout *layout, bool is_cuda) {
```

Comment:
any particular reason for this reordering? (did anything change?)

Reply:
calling it in initialize_python_bindings now